.gitignore

```diff
@@ -16,6 +16,7 @@ crash.log
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
 *plan*
+!src/bicep/**
```

Comment:
Is this change intentional?

Reply:
Yes sir.   The !src/bicep/** creates an exception to the preceding line before it, "*plan*".  The exception is required because the name of the Azure service I am trying to implement is "App Service Plan".  Git was ignoring all the files in this commit because of that line "*plan*".  Make sense?

The line also is for terraform, so I just excepted the entire bicep directory and all subdirectories.   I did double check, no files other than mine were being affected but it seems like it would save future devs from having to do it.